src/index.css

```diff
@@ -378,7 +378,7 @@ html {
 
 body {
   color: #404040;
-  background: white;
+  background: rgb(221, 228, 247);
```

Comment:
Good color selection, page is looking pretty

Reply:
Thanks. It took a while to find a color scheme that worked well together